payjoin/src/uri.rs

```diff
@@ -274,11 +302,67 @@ enum InternalPjParseError {
     UnsecureEndpoint,
 }
 
+/// Utility for a receiving wallet to build a payjoin URI
+pub struct PjUriBuilder<'a> {
+    address: Address,
+    pj_endpoint: Url,
+    amount: Option<Amount>,
+    ohttp: Option<&'a str>,
```

Comment:
this should probably be the validated `ohttp::KeyConfig` type conditionally compiled for `#[cfg(feature = "v2")]`